test/test_runner.go

```diff
@@ -722,7 +691,30 @@ func (r *TestRunner) parseAndCheckImport(
 			}, nil
 
 		default:
-			return nil, fmt.Errorf("nested imports are not supported")
+			addressLoc, ok := importedLocation.(common.AddressLocation)
+			if ok {
+				// System-defined contracts are obtained from
+				// the blockchain.
+				account, err := r.blockchain.GetAccount(
+					flow.Address(addressLoc.Address),
+				)
+				if err != nil {
+					panic(err)
+				}
```

Comment:
same here

Reply:
Good catch :+1: 
Fixed in https://github.com/onflow/cadence-tools/pull/205/commits/3b60ad4a5f96051465b9954f9aa0f5ebbf2eb794